gpkit/tools/autosweep.py

```diff
@@ -126,6 +138,27 @@ def sample_at(self, values):
         "Creates a SolutionOracle at a given range of values"
         return SolutionOracle(self, values)
 
+    @property
+    def sollist(self):
+        "Returns a list of all the solutions in an autosweep"
+        if self.splits:
+            return [self.sols[0], self.splits[0].sollist[1:-1],
+                    self.splits[1].sollist[:-1], self.sols[1]]
```

Comment:
I flatten it in solarray, but thought maybe it'd be good to preserve structure here?